.gitlab-ci.yml

```diff
@@ -213,5 +216,21 @@ deploy-production:
   tags:
     - kubernetes-parity-build
   only:
+    - master
     - tags
+    - /^v[0-9]+\.[0-9]+\.[0-9]+.*$/
   <<: *deploy-k8s
+
+publish-draft-release:
+  stage: publish
+  environment:
+    name: parity-prod
+  image: parity/tools:latest
+  only:
+    - master
```

Comment:
This will cause a draft release to be created every time there is a push to master - that's likely not what you want

Reply:
Ah I though this was an "and", like if it's a tag, with vx.y.z and on master. Am I wrong?

Reply:
Thanks, I've seen this "except" quite often, and it always feels wrong to me. I'll keep it simple indeed, just the regex will be fine.